### ISSUE TITLE
Some C code segments are difficult to read in dark mode

### ISSUE THREAD

**Issue by nilsreichardt** [Jwrede/Anki-KaTeX-Markdown#45]
<img width="950" alt="image" src="https://github.com/Jwrede/Anki-KaTeX-Markdown/assets/24459435/5f209aae-708b-47dd-be36-1ef845902b17">

I have the dark mode enabled in Anki. When I'm writing C code, some of the code segments are difficult to read (see screenshot):

Example code:

```c
#include <stdlib.h>
#include <string.h>

int main() {
    // Allocate memory for 10 characters on the heap
    char *p = (char*) malloc(10 * sizeof(char));

    // Copy more than 10 characters into the allocated memory.
    // This results in a heap-buffer-overflow error.
    strcpy(p, "This string is too long for the allocated buffer");

    // Don't forget to free your memory
    free(p);

    return 0;
}
```

**Comment by alexthillen** [comment]
You can try my fork : [Anki add on](https://ankiweb.net/shared/info/1786114227) ; [Github Fork](https://github.com/alexthillen/Anki-KaTeX-Markdown/releases/tag/release%2F7)
It adds a new card type (KaTex and Markdown Basic (Color))
It will look like:
<img width="1446" alt="image" src="https://github.com/Jwrede/Anki-KaTeX-Markdown/assets/21108667/fbb00953-89fd-4d18-9b87-f292b26e1f24">

**Comment by EngineNeo** [comment]
Thanks for the fork. While it helped with post-card creation issues, it did not fix card creation issues where the text is borderline unreadable without turning on light mode.

![image](https://github.com/Jwrede/Anki-KaTeX-Markdown/assets/10225727/f5f8dc57-fb56-4443-8abc-87bbc2b3c47c)

**Comment by alexthillen** [comment]
yes you are right, might take a look at card creation as well. Also cloze is not supported inside code block at this moment unfortunately :/

**Comment by alexthillen** [comment]
In case you are still interested, it's working fine now.
<img width="548" alt="image" src="https://github.com/Jwrede/Anki-KaTeX-Markdown/assets/21108667/26cb28c3-f387-4ca4-a97f-cfba489075b7">
